crc/aarch64/crc64_rocksoft_norm_pmull.h

```diff
@@ -0,0 +1,210 @@
+########################################################################
+#  Copyright(c) 2025 Tim Burke All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions
+#  are met:
+#    * Redistributions of source code must retain the above copyright
+#      notice, this list of conditions and the following disclaimer.
+#    * Redistributions in binary form must reproduce the above copyright
+#      notice, this list of conditions and the following disclaimer in
+#      the documentation and/or other materials provided with the
+#      distribution.
+#    * Neither the name of Arm Corporation nor the names of its
+#      contributors may be used to endorse or promote products derived
+#      from this software without specific prior written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+#  "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+#  LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+#  A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+#  OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+#  SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+#  LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+#  DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+#  THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+#  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+#  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#########################################################################
+
+# rk15
+.equ	p4_low_b0, 0x488c
+.equ	p4_low_b1, 0x0488
+.equ	p4_low_b2, 0x4e6a
+.equ	p4_low_b3, 0xb441
+# rk16
+.equ	p4_high_b0, 0x9860
+.equ	p4_high_b1, 0x9b66
+.equ	p4_high_b2, 0x30f1
+.equ	p4_high_b3, 0xa42a
+
+# rk1
+.equ	p1_low_b0, 0x2f08
+.equ	p1_low_b1, 0xf0dd
+.equ	p1_low_b2, 0xc948
+.equ	p1_low_b3, 0x6b08
+# rk2
+.equ	p1_high_b0, 0x76ae
+.equ	p1_high_b1, 0x7f04
+.equ	p1_high_b2, 0x8ba9
+.equ	p1_high_b3, 0x0857
+
+# rk1
+.equ	p0_low_b0, 0x2f08
+.equ	p0_low_b1, 0xf0dd
+.equ	p0_low_b2, 0xc948
+.equ	p0_low_b3, 0x6b08
+.equ	p0_high_b0, 0x0000
+.equ	p0_high_b1, 0x0000
+.equ	p0_high_b2, 0x0000
+.equ	p0_high_b3, 0x0000
+
+# rk7
+.equ	br_low_b0, 0x6fc8
+.equ	br_low_b1, 0x98be
+.equ	br_low_b2, 0xeeb2
+.equ	br_low_b3, 0xddf3
+# rk8
+.equ	br_high_b0, 0x3659
+.equ	br_high_b1, 0x94c9
+.equ	br_high_b2, 0xd235
+.equ	br_high_b3, 0xad93
+
+	.text
```

Comment:
It seems like this .text directive shouldn't be here, as this section is declaring read only data, not executable code.

Reply:
Dropped, here and from the others.